static/app/utils/analytics/signupAnalyticsEvents.tsx

```diff
@@ -0,0 +1,12 @@
+export type SignupAnalyticsParameters = {
+  'join_request.created': {
+    organization: string;
```

Comment:
organization doesn't need to be a parameter here since it's needed, you only need to include the additional ones.